exo/inference/tinygrad/models/llama.py

```
@@ -180,6 +230,7 @@ def __init__(
     vocab_size,
     shard: Shard = None,
     linear=nn.Linear,
+    embedding=nn.Embedding,
```

Ruff (0.8.2)

Missing type annotation for function argument `embedding`

(ANN001)